src/components/forms/event-form.js

```diff
@@ -444,7 +445,12 @@ class EventForm extends React.Component {
   handleMaterialDownload(materialId) {
     const { entity } = this.props;
     const material = entity.materials.find((m) => m.id === materialId);
-    window.open(material.private_url || material.public_url, "_blank");
+    window.open(
```

Comment:
this is not a correct approach
u should get a valid url first beofre open the windows
or display an error
const url = isValidUrl(material.private_url) ? material.private_url : (isValidUrl(material.public_url) ? material.public_url : null)
if(!url) throw error